### PR TITLE
adjusts the mlir attrs order, test=develop

### PR DESCRIPTION
**Pull request by Shixiaowei02**
### PR types

Bug fixes
### PR changes

Others
### Describe

adjusts the mlir attrs order, test=develop

### CODE REVIEW

File: paddle/infrt/host_context/mlir_to_runtime_translate.cc [hunk 1]
```diff
@@ -41,6 +41,7 @@
 #include "paddle/infrt/host_context/op_executable.h"
 #include "paddle/infrt/host_context/value.h"
 #include "paddle/infrt/tensor/tensor_shape.h"
+#include "paddle/phi/core/enforce.h"
```
Comment (jiweibo) [inline paddle/infrt/host_context/mlir_to_runtime_translate.cc]: phi是否需要用INFRT_WITH_PHI隔离

**Review by jiweibo — DISMISSED** [review]
LGTM

File: paddle/infrt/host_context/mlir_to_runtime_translate.cc (hunk 1 above)
Comment (Shixiaowei02) [reply paddle/infrt/host_context/mlir_to_runtime_translate.cc]: done

**Review by winter-wang — APPROVED** [review]
LGTM

**Pull request merged**